src/brogue/Recordings.c

```diff
@@ -726,6 +726,205 @@ void pausePlayback() {
     }
 }
 
+// Pass in defaultPath (the file name WITHOUT suffix), and the suffix.
```

Comment:
This would be a lot easier to review without the moving around of all these functions which haven't changed.

Reply:
You're right, there was a much simpler fix for what I wanted.